internal/apitest/runner.go

```diff
@@ -425,6 +425,7 @@ func (r *Runner) runSuites() bool {
 			Str("projectName", suite.ProjectName).
 			Str("suite", suite.Name).
 			Bool("parallel", true).
+			Str("tunnelName", r.Project.Sauce.Tunnel.Name).
```

Comment:
Maybe we can shorten the field name? Saves a bit of real estate in the actual log line and is still clear enough I think:
```suggestion
			Str("tunnel", r.Project.Sauce.Tunnel.Name).
```

(also apply to all other log statements in this PR)